README.md

```diff
@@ -101,17 +101,6 @@ Getting Started
 
 Execute `cibuild.cmd` to clean, restore, build and runs tests
 
-**NOTE** The current build of System.Reflection.Metadata (from NuGet package System.Reflection.Metadata.1.2.0-rc2-23629) is unsigned. This causes unit tests to fail. To work around this problem until a new, signed version is available, give the command
```

Comment:
Can't comment on the specific line, but the getting started instructions above this need to be updated.